README.md

```diff
@@ -40,43 +40,48 @@ To configure the library for use from a client (ie. a service that needs to obta
 the following configuration is supported:
 
 ```elixir
-config :prima_auth0_ex,
+config :prima_auth0_ex, :client,
   # Base url for Auth0 API
   auth0_base_url: "https://tenant.eu.auth0.com"
-
-config :prima_auth0_ex, :client,
   # Credentials on Auth0
   client_id: "",
   client_secret: "",
-  # Enables cache on redis for tokens obtained from Auth0. Defaults to true.
-  cache_enabled: true,
-  # Namespace for tokens of this service on the shared cache. Should be unique per service (e.g., the service name)
-  cache_namespace: "my-service",
-  # AES 256 key used to encrypt tokens on the shared cache.
-  # Can be generated via `:crypto.strong_rand_bytes(32) |> Base.encode64()`.
-  cache_encryption_key: "uhOrqKvUi9gHnmwr60P2E1hiCSD2dtXK1i6dqkU4RTA=",
-  redis_connection_uri: "redis://redis:6379",
-  redis_ssl_enabled: false,
-  # Read here for more infos: https://hexdocs.pm/redix/Redix.html#module-ssl
-  redis_ssl_allow_wildcard_certificates: false
+  # Namespace for tokens of this client on the shared cache. Should be unique per client
+  cache_namespace: "my-client",
 ```
 
 **If the client will access APIs that perform validation of permissions, make sure that the API on Auth0 is configured to have both "Enable RBAC" and "Add Permissions in the Access Token" enabled.**
 Otherwise, the JWTs generated by Auth0 may not include the necessary permissions claims.
 
 A visualization of the logic behind the `TokenProvider` is available [here](client_flow.jpg).
 
-### API Provider
+#### Multiple clients
 
-To configure the library for use from a server (ie. a service that exposes an API),
-the following configuration is supported:
+If you need to use multiple clients which target different Auth0 instances, you can configure them like so
 
 ```elixir
-config :prima_auth0_ex,
+config :prima_auth0_ex, :clients, client_name: [
```

Comment:
Is it possible to make it clearer that this `client_name` is chosen by the user and is not a proper configuration key? Maybe changing it in `your_client_name` or `{client_name}:` or whatever you feel is better

Reply:
Updated it to make it clearer